.github/ISSUE_TEMPLATE/code_first.yml

```diff
@@ -0,0 +1,161 @@
+# TianoCore edk2 GitHub Code First Template
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+name: </> Code First
+description: Code first tracking issue
+title: "[Code First]: <title>"
+labels: ["type:code-first"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        👋 Only use this issue form for changes following the "code first" process described in [EDK II Code First Process](https://github.com/tianocore/tianocore.github.io/wiki/EDK-II-Code-First-Process).
+
+        **Read that document before filing this issue.**
+
+  - type: textarea
+    id: overview
+    attributes:
+      label: Code First Item Overview
+      description: Provide a brief overview of the overall code first change.
+    validations:
+      required: true
+
+  - type: dropdown
+    id: specs_impacted
+    attributes:
+      label: What specification(s) are directly related?
+      description: |
+        *Select all that apply*
+      multiple: true
+      options:
+        - ACPI
+        - Platform Initialization (PI)
+        - UEFI
+        - UEFI PI Distribution Packaging
+        - UEFI Shell
+    validations:
+      required: true
+
+  - type: textarea

```

Comment:
Does this mean that the text of the spec change is in the GitHub issue description and that the markdown file with the spec change is no longer required?

Reply:
You're referring to the copy that is in https://github.com/tianocore/tianocore.github.io/wiki/EDK-II-Code-First-Process as the markdown file, right?

Reply:
yes

Reply:
Yes. I'd be in favor of dropping it from there. I will make that update in the pending changes in https://github.com/makubacki/tianocore.github.io/pull/3.

Reply:
Updated this change to include some additional instructions from the wiki page now that the template is centralized here.

Reply:
I was not following the template syntax and could not tell if the markdown content was intended to be added to the issue or just just there for reference to show the markdown required sections.  I do not think putting the text of the actual spec change in the issue is the best idea.  I think that should be under change control with git to be able to review history of code first spec changes.  Same for the code first implementation work.

Reply:
So you'd like to just have the markdown for reference in the issue with instructions to include the markdown file with the completed template in the code changes and remove the textbox from the form?

Reply:
Yes.  I do not think you can comment on the issue description and resolve conversations here.  We really want code review feature for the spec changes.

Reply:
Got it. Update is pushed.